net/netns/netns_darwin.go

```diff
@@ -25,6 +29,10 @@ func control(logf logger.Logf) func(network, address string, c syscall.RawConn)
 	}
 }
 
+var bindToDeviceByRoute = envknob.RegisterBool("TS_BIND_TO_DEVICE_BY_ROUTE")
```

Comment:
Environment variables are annoying to set on macOS and ~impossible on iOS. Should we also add a node attribute that can be used to enable this?

Reply:
Yep, good idea; will add.

Reply:
Added 👍

Reply:
Filed https://github.com/tailscale/tailscale/issues/7077 too now that you reminded me :)